docs/use/vscode.md

```diff
@@ -513,6 +513,53 @@ If multiple terminals are shared, or your focus is in a different tab, you can b
+
+### General information
+
+At all times, Live Share will
+
+- Play a sound when you follow someone and announce their name and position.
+- Play a sound when you unfollow someone and announce their name.
+- Announce when a guest joins or leaves the session.
+
+### Outside of follow mode
+
+When you are not following another participant
+
+- If someone edits on your line, Live Share will play an edit sound and announce the name of the participant editing on your line. For subsequent edits, Live Share will play the edit sound with no announcement. Live Share will announce when they leave your line.
+- If someone edits nearby, Live Share will play an edit sound and announce the name of the participant that is editing nearby. For subsequent edits, Live Share will play a sound indicating the participant is editing nearby with volume proportional to how near they are.
+
+### Follow mode
+
+When you are following another participant, Live Share provides you more information about their activity. Live Share will
+
+- Play an edit sound when they type.
+- Play a sound when they change position due to a command, such as Go to Definition. 
+- Play a sound when they change position via the keyboard. For a single line change, Live Share will play a key click, followed by a bend sound that indicates the direction of the change, followed by an announcement of the new line. For a multi-line change, Live Share will play a bend sound that indicates the direction of the change, followed by an announcement of the new line.
+- Play a bend sound when they change position via the keyboard that indicates the direction of the change, followed by an announcement of the new line.
+- Announce that they made a selection. For single line selections, Live Share will read the selection contents. For multi-line selections, Live Share will announce the selected lines.
+- Announce when they switch files, indicating the new file name.
+- Play a scrolling sound when they scroll, followed by a bend sound that indicates the direction of the change, followed by an announcement of the new visible range.
+
+### Customization
+
+Live Share provides several additional settings to customize these audio cues.
+
+- `liveshare.accessibility.volume` - sets the volume of sounds and announcements.
+- `liveshare.accessibility.prosodyRate` - sets the prosody of announcements.
+- `liveshare.accessibility.voice` - sets the voice used for announcements. Currently supports 37 different voices, spanning genders, ages, and accents.
+
+Live Share also contributes a command `liveshare.accessibility.setOutputDevice` that allows you to change the output device used to play audio cues.
+
+> This command is currently only available on Windows.
+
+### Cursor tethering
+
+With accessibility features enabled, when following another participant Live Share will keep your cursor locked to the same position that the followed participant's cursor is at to make it easier to use the screen reader.
+
```

Comment:
If you would like to provide any feedback on our accessibility features, please file at issue here: https://github.com/MicrosoftDocs/live-share/issues/new?assignees=&labels=&template=feature_request.md

Reply:
We don't call out where to leave feedback anywhere else in the documentation? I don't know why accessibility would warrant a call-out over any of the other features discussed in the docs.

Reply:
Because its a new feature that is more sensitive that others? More inclusive? Or do you think its worse to do that?

Reply:
I just think it's kind of strange. People should leave feedback for *any* part of the product that doesn't work. Seems best to have one section about filing feedback that applies to the entire document rather than each section calling out how to leave feedback. Basically it's documentation that applies to the entire product, not just this feature.